src/components/FieldCheckbox.tsx

```diff
@@ -12,53 +12,56 @@ import Touchable from "./Touchable";
 import Checkbox from "./Checkbox";
 
 import color from "color";
+import theme from "../styles/DefaultTheme";
 
-class FieldCheckbox extends React.Component {
-  onPress = () => {
-    const { onPress } = this.props;
-    if (onPress) {
-      onPress();
-    }
-  };
-
-  render() {
-    const {
-      title,
-      status,
-      onPress,
-      color: checkboxColor,
-      disabled,
-      theme: { colors, typography, spacing, disabledOpacity },
-    } = this.props;
-
-    let titleColor = status === "checked" ? colors.medium : colors.light;
+interface Props {
+  title?: string;
+  status: "checked" | "indeterminate" | "unchecked";
+  onPress?: () => void;
+  color: string;
+  disabled?: boolean;
+  theme: typeof theme;
+}
 
-    if (disabled) {
-      titleColor = color(titleColor).alpha(disabledOpacity).rgb().string();
-    }
+const FieldCheckbox: React.FC<Props> = ({
+  title = "",
```

Comment:
```suggestion
  title,
```

will this break if its not a string?

Reply:
No, just tested it. I works fine when without the string.